CONTRIBUTING.md

```diff
@@ -26,9 +29,63 @@ The Corporate CLA is for those who are contributing as a member of an organizati
 - [CLA for individuals](https://opensource.atlassian.com/individual)
 - [CLA for corporate contributors](https://opensource.atlassian.com/corporate)
 
+## Key Concepts
+
+### CSS-in-JS
+
+Compiled is a CSS-in-JS library: a library that lets frontend developers write CSS within their JavaScript code. Other libraries in this space include styled-components and Emotion. Below are some code examples.
+
+How people usually write in Compiled (we do want to change this long-term). This is similar to the styled function from [styled-components](https://styled-components.com/).
```

Comment:
Thoughts on just provide a  link to the [website](https://compiledcssinjs.com/docs/) and make sure that the related sections are up-to-date? It may be easier to update it in the future, if we need to maintain the info only in 1 place

Reply:
hmm perhaps https://compiledcssinjs.com/docs/writing-css could be used for this

(https://github.com/compiled/website/blob/master/packages/docs/src/pages/writing-css.mdx)